src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -234,12 +237,14 @@ public ExpiredDocumentsCleaner.DeleteExpiredDocumentsCommand ToCommand(Documents
             {
                 expired[item.Key] = item.Value;
             }
-            var command = new ExpiredDocumentsCleaner.DeleteExpiredDocumentsCommand(expired, database, ForExpiration);
+            var command = new ExpiredDocumentsCleaner.DeleteExpiredDocumentsCommand(expired, database, ForExpiration, CurrentTime);
             return command;
         }
 
+        public KeyValuePair<Slice, List<(Slice LowerId, string Id)>>[] Expired { get; set; }
+
         public bool ForExpiration { get; set; }
 
-        public KeyValuePair<Slice, List<(Slice LowerId, string Id)>>[] Expired { get; set; }
+        public DateTime CurrentTime { get; set; }
```

Comment:
Who sets this value?

Reply:
It is set during deserialization in ReplayCommandTxHelper